content/en/docs/devops-user-guide/examples/customize-jenkins-agent.md

```diff
@@ -43,4 +43,28 @@ This document describes how to customize a Jenkins agent on KubeSphere.
 
 6. Wait for at least 70 seconds until your changes are automatically reloaded.
 
-7. To use the custom Jenkins agent, specify the label and container name of the custom Jenkins agent accordingly when creating a pipeline.
+7. To use the custom Jenkins agent, refer to the following sample Jenkinsfile to specify the label and container name of the custom Jenkins agent accordingly when creating a pipeline.
+
+   ```groovy
+   pipeline {
+     agent {
+       node {
+         label 'maven jdk11'
```

Comment:
Please note that the label should be set to `maven && jdk11`.

Reply:
Actually, users can use it with the following pattern:

* `maven`
* `jdk11
* `maven || jdk11`
* `maven && jdk11`